CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# main
+## Features ✨ and improvements 🏁
+* Introduce ModelLayer experimental API to render 3D model on the map. ([#1369](https://github.com/mapbox/mapbox-maps-android/pull/1369))
```

Comment:
```suggestion
* Introduce experimental Model API to render 3D models on the map. ([#1369](https://github.com/mapbox/mapbox-maps-android/pull/1369))
```
I'd avoid stating `ModelLayer` as we do also introduce `model` classes as well but up to you